commands/create/cluster/command.go

```diff
@@ -193,14 +203,15 @@ func initFlags() {
 	Command.Flags().StringVarP(&flags.ClusterName, "name", "n", "", "Cluster name")
 	Command.Flags().StringVarP(&flags.Owner, "owner", "o", "", "Organization to own the cluster")
 	Command.Flags().StringVarP(&flags.Release, "release", "r", "", "Release version to use, e. g. '1.2.3'. Defaults to the latest. See 'gsctl list releases --help' for details.")
+	Command.Flags().BoolVar(&cmdMasterHA, "master-ha", true, "This means the cluster will have three master nodes. Requires High-Availability Master support.")
```

Comment:
I think `cmdMasterHA` should be defined in the `flags` package, just like the others.